src/pyfaf/actions/attach_centos_bugs.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2015  ABRT Team
+# Copyright (C) 2015  Red Hat, Inc.
+#
+# This file is part of faf.
+#
+# faf is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# faf is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with faf.  If not, see <http://www.gnu.org/licenses/>.
+
+from pyfaf.actions import Action
+from pyfaf.bugtrackers import bugtrackers
+from pyfaf.queries import (get_mantis_bug,
+                           get_report_by_hash,
+                           get_reportmantis,
+                           get_osrelease)
+from pyfaf.storage import ReportMantis
+
+
+class AttachCentosBugs(Action):
+    """
+    Looks up all bugs created using ABRT on CentOS MantisBT and attaches them
+    to reports using the bthash from URL. This is a workaround until proper
+    attaching is supported by the ABRT client.
```

Comment:
> So you think we are idiots and we didn't implemented the attaching just for fun?

No I'm saying you've deployed an incomplete feature and you're doing this over and over again.

> Why are you bringing this up? There were several test days of the CentOS integration and I don't remember you taking part of any.

I prefer automation and I've invested a LOT of time into it so I don't have to do things like test days..

> We know that the integration is incomplete and we don't call it a feature. We just got a more priority task.

Like what? Adding new features?

> We do what users request and managers order. Feel free to step up and become a leader.

Maybe use your head as well from time to time.